tests/Unit/DataStructures/VectorImplTestHelper.hpp

```diff
@@ -446,7 +446,9 @@ struct VectorOrArraySize {
   }
   // vector version
   template <typename T,
-            Requires<cpp17::is_arithmetic_v<typename T::ElementType>> = nullptr>
+            Requires<cpp17::is_arithmetic_v<typename T::ElementType> or
```

Comment:
are these changes tested anywhere?

Reply:
I've added a test for the custom size and element functions in `Test_VectorImplTestHelper.cpp`. For the order of the commits, the fixup to the utilities commit includes only tests of `DataVector`, and the additional tests for `ComplexDataVector` are added in the fixup commit for implementing `ComplexDataVector`.